architect/systems/optical/spectrometers.py

```diff
@@ -303,14 +303,13 @@ def get_pointing_accuracy_constraint(
 
         return constraint_angle
 
-    def get_ground_target_error(self, orbital_altitude, skew_angle):
+    def get_ground_target_error(self, orbital_altitude, skew_angle, pointing_accuracy: unit.deg = 0):
         """Get the ground target error from the skew angle error.
```

Comment:
Update dosctring to match function arguments
```suggestion
        """Get the ground target error from the pointing accuracy.
```